search.js

```diff
@@ -28,13 +28,30 @@ const {argv} = require('yargs')
   choices: ['modules','settings'], 
   description: 'Field type to look up'
 })
+.option('timeout', {
+  type: 'int',
+  default: 1000,
+  description: 'Timeout per request, in ms; 0 means no timeout'
+})
+.option('cursor', {
+  type: 'string',
+  description: 'A cursor to start querying with'
+})
+.option('live', {
+  type: Boolean,
+  default: false,
+  description: 'If true, output results one page at a time; helpful for debugging'
+})
```

Comment:
```suggestion
})
.boolean('live')
```
per https://yargs.js.org/docs/#api-reference-booleankey